app/components/Application/ApplicationOverrideNotificationCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Card} from 'react-bootstrap';
+
+interface Props {
+  overrideJustification: string;
+}
+
+const ApplicationOverrideNotification: React.FunctionComponent<Props> = ({
+  overrideJustification
+}) => {
+  return (
+    <Card style={{margin: '1rem 0'}}>
+      <Card.Header className="bg-danger">
+        <h4>Override is active</h4>
+      </Card.Header>
+      <Card.Body>
+        <Card.Text>
+          <strong>
+            There were errors in this application that the reporter has chosen
+            to override. Justification for overriding the errors is provided in
+            the box below:
+          </strong>
+        </Card.Text>
+        <Card>
+          <div tabIndex={0} id="justification" className="scrollbar">
```

Comment:
For keyboard scrolling, `tabindex` must to be set directly on the content element that is overflowing its container. In this case, it should be moved to the `<Card.Body tabindex="0">` instead of the `<div id="justification">`